nerfstudio/data/dataparsers/colmap_dataparser.py

```diff
@@ -56,6 +56,8 @@ class ColmapDataParserConfig(DataParserConfig):
     """How much to scale the camera origins by."""
     downscale_factor: Optional[int] = None
     """How much to downscale images. If not set, images are chosen such that the max dimension is <1600px."""
+    downscale_rounding_mode: Literal["floor", "ceil"] = "floor"
```

Comment:
The ceiling mode isn't compatible with the ffmpeg commands. 

See here: https://github.com/nerfstudio-project/nerfstudio/blob/main/nerfstudio/data/dataparsers/colmap_dataparser.py#L464-L468

You probably need to change this part and test the auto-downscale implementation.

Reply:
Hi @jb-ye, I've changed this part. I ran 'ns-train nerfacto --data xxx colmap --downscale_rounding_mode round --downscale_factor 4' and it worked when there only original image folder and colmap folder under the data path. Do you have any other suggestions:)?